src/less/carousel/base/carousel.less

```diff
@@ -68,6 +68,20 @@
     }
 }
 
+.carousel__container--image-treatment {
+    .carousel__list img {
+        display: inline-block;
+        object-fit: contain;
+        max-width: 100%;
+    }
+
+    .carousel__list li {
+        background-color: rgba(0, 0, 0, 0.05);
```

Comment:
Is there a way we can do opacity only with background-color, or at least use `opacity`? Does it work or does it also add opacity to the image?
I'm fine with it this way, but wanted to see if we can do it another way or if it was tried.

Reply:
Adding opacity also affects the image in this case.